src/main/resources/swagger/cromwell.yaml

```diff
@@ -604,6 +604,9 @@ definitions:
       stderr:
         type: string
         description: Path to the standard error file for this call
+      backendLogs:
+        type: object
```

Comment:
I mean, what's in one? As an API user, what should I expect to get back when I make this call?